test/component/storage/store_test.go

```diff
@@ -0,0 +1,376 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package storage_test
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	drivermysql "github.com/go-sql-driver/mysql"
+	"github.com/google/uuid"
+	"github.com/hyperledger/aries-framework-go/pkg/storage"
+	dctest "github.com/ory/dockertest/v3"
+	dc "github.com/ory/dockertest/v3/docker"
+	"github.com/stretchr/testify/require"
+
+	"github.com/hyperledger/aries-framework-go-ext/component/storage/couchdb"
+	"github.com/hyperledger/aries-framework-go-ext/component/storage/mysql"
+)
+
+const (
+	couchDBURL          = "admin:password@127.0.0.1:5983"
+	dockerCouchdbImage  = "couchdb"
+	dockerCouchdbTag    = "3.1.0"
+	dockerCouchdbVolume = "%s/component/storage/couchdb/testdata/single-node.ini:/opt/couchdb/etc/local.d/single-node.ini"
+
+	sqlStoreDBURL    = "root:my-secret-pw@tcp(127.0.0.1:3300)/"
```

Comment:
I would prefer not to mix modules in the same file. We want this to be maintainable by the party responsible for that module.